privacy-security-explainer.md

```diff
@@ -3,7 +3,26 @@
 
 ### Sensitive information
-In the context of XR, sensitive information includes, but is not limited to, user configurable data such as interpupillary distance (IPD) and sensor-based data such as poses. All `immersive` sessions will expose some amount of sensitive data, due to the user's pose being necessary to render anything. However, in some cases, the same sensitive information will also be exposed via `inline` sessions. 
+In the context of XR, sensitive information includes, but is not limited to, user configurable data such as interpupillary distance (IPD) and sensor-based data such as poses. All `immersive` sessions will expose some amount of sensitive data, due to the user's pose being necessary to render anything. However, in some cases, the same sensitive information will also be exposed via `inline` sessions.
+
+### Device Fingerprinting
+The user agent must take steps (where possible) to prevent exposing data that is unique to the device. This is true even if consent has been obtained for other reasons. For example, even if user consent is obtained to expose IPD data (for reasons of user profiling) it is still in the user's best interests that the IPD data be anonymized to prevent fingerprinting.
+
+Some raw sensor data might be fingerprinted (e.g. IMU data) and thus on some user agents access to that raw sensor data outside of WebXR requires user consent. In cases where the same sensor fingerprinting risk exists for WebXR data, the user agent must either anonymize the data sufficiently to prevent the risk, otherwise [explicit consent](#explicit-consent) is strongly recommended.
+
+Specific approaches to mitigating device fingerprinting are up to the user agent who is best equipped to evaluate the actual threat on a given platform using the platform's APIs.
+
+### User Profiling
+This explainer prioritizes highly the protection of sensitive user characteristics. If there is a reasonable possibility that a reliable signal for a sensitive characteristic exists for some population of users, then [explicit consent](#explicit-consent) is strongly recommended. For example, [explicit consent](#explicit-consent) is strongly recommended before exposing data that sites might use to reliably infer sensitive user characteristics such as race, gender, or age. 
```

Comment:
"sensitive characteristics" could probably use a stronger, concrete definition.  Perhaps in the same initial paragraph regarding "sensitive data"?

Reply:
I've updated this to refer to 'demographic' characteristics instead. WDYT?